federation/pkg/kubefed/util/util.go

```diff
@@ -150,3 +156,57 @@ func CreateKubeconfigSecret(clientset *client.Clientset, kubeconfig *clientcmdap
 	}
 	return secret, nil
 }
+
+func WaitForLoadBalancerAddress(clientset *client.Clientset, svc *api.Service) ([]string, []string, error) {
```

Comment:
(No action required) No tests for either of these functions (and hence no test changes required).